cpp/include/ray/api.h

```diff
@@ -39,6 +39,14 @@ class Ray {
   template <typename T>
   static ObjectRef<T> Put(const T &obj);
 
+  /// Get a single object from the object store.
+  /// This method will be blocked until all the objects are ready.
```

Comment:
But you are passing a single object right?
```suggestion
  /// This method will be blocked until the object is ready.
```